.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Add a team or username to this file
+# Example:
+# *   @ministryofjustice/operations-engineering
```

Comment:
should this have a team mentioned in this file?

Reply:
It's commented out and is an example. So, yes it should once a team has a clone of this repo.